test/test_devices.py

```diff
@@ -1,6 +1,11 @@
 from unittest import TestCase
 from mock import Mock, MagicMock, patch
 from serial import Serial, SerialException
+try:
```

Comment:
Looks like a little more than a cleanup for merge. Please explain the reason for this change.